telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -150,8 +129,7 @@ const DashboardTodo = () => {
     setOpen(false);
   };
 
-  // Adding a todo from the 'New to do' dialog
```

Comment:
Was this comment meant to be removed? seems worth keeping.

Reply:
Added that back